libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationView.kt

```diff
@@ -104,8 +107,15 @@ class DropInNavigationView @JvmOverloads constructor(
     }
 
     init {
+        val style = if (isNightModeEnabled()) {
+            NavigationStyles.NAVIGATION_NIGHT_STYLE
+        } else {
+            NavigationStyles.NAVIGATION_DAY_STYLE
+        }
+        binding.mapView.compass.enabled = false
+        binding.mapView.scalebar.enabled = false
```

Comment:
Would like to move all of this away from the DropInNavigationView. 

Can this be moved into  `MapCoordinator.onAttached`.. maybe even `MapCoordinator.init`

Reply:
Done.